packages/editor/src/core/components/Editable/Inner/index.tsx

```diff
@@ -84,10 +85,16 @@ const Inner: React.FC = () => {
       style={{ minHeight: 480 }}
       className={'react-page-editable react-page-editable-mode-' + mode}
     >
-      {rowIds.length > 0
-        ? rowIds.map((id) => <Row nodeId={id} key={id} />)
-        : null}
-      {insertAllowed ? <InsertNew /> : null}
+      <div
+        style={{
+          marginBottom: mode !== 'preview' ? null : `${-cellSpacing}px`,
```

Comment:
why is the spacing removed on preview mode?

Reply:
Oops. I though I fixed that. The idea is simply to exclude the InsertNew component from the compensation box.
The right way of doing that is the same as in the Inner cell: wrap a compensating div around the children (if any) and exclude InsertNew.